CHANGELOG.md

```diff
@@ -14,7 +14,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [0.0.2] - 2019-05-31
 ### Added
-- Navigation footer
+-  Changes to the Wizard navigation footer and other DAVE related buttons
```

Comment:
@erikadionn we just want to move the new item into the "unreleased" section. That navigation footer item should stay, as it expresses the navigation footer you added previously. When that's done, this is good to merge.